site/theme/template/Content/ComponentDoc.jsx

```diff
@@ -52,7 +52,13 @@ class ComponentDoc extends React.Component {
   };
 
   render() {
-    const { doc, location, intl: { locale }, utils, demos } = this.props;
+    const {
+      doc,
+      location,
+      intl: { locale },
+      utils,
+      demos,
+    } = this.props;
```

Comment:
Mixed some unrelated changes

Reply:
I can't exclude these files.
I ran this command and fetched upstream file.
`git checkout upstream/feature -- site/theme/template/Content/Demo.jsx`
but after commit and push files won't disappear from changes.

Reply:
Just fixed issue by editing file from browser and Github web. :|
I think on my local system it auto-format the file after commit and re-arranges the render method parameters.